admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -736,7 +736,9 @@ mail_domain
 
 The return address that you want to appear on emails sent by the Nextcloud
 server, for example ``nc-admin@example.com``, substituting your own domain,
-of course.
+of course. Please note that this is *not* the domain used for the SMTP protocol handhshake.
+So if you want to have the server responding with an EHLO reply identifiying itself properly,
+please configure ``overwrite.cli.url`` accordingly.
```

Comment:
I'd rather say that it uses the value of ``overwrite.cli.url``. Because like that, you might get the impression to set this variable accordingly to make the mail delivery work. But it has other consequences as well (it's not use exclusively for the mail EHLO).